tests/ui/lang/core/ref/member_ref_arg-broken.stderr

```diff
@@ -3,6 +3,7 @@ warning: `#[inline(never)]` function `member_ref_arg_broken::h` needs to be inli
 warning: `#[inline(never)]` function `member_ref_arg_broken::h_newtyped` needs to be inlined because it has illegal argument or return types
 
 error: error:0:0 - OpLoad Pointer <id> '$ID[%$ID]' is not a logical pointer.
+         %39 = OpLoad %uint %38
```

Comment:
Love to finally have this resolved! We still should clean up some of the formatting so it's easier to understand that this is a `spirv-val` error, but at least now we won't have test output differ between "compiled vs installed" tools.